test/dataframe.jl

```diff
@@ -377,6 +429,36 @@ module TestDataFrame
         a = unstack(df, :id, :variable, :value)
         b = unstack(df, :variable, :value)
         @test a ≅ b ≅ DataFrame(id = [1, 2], a = [3, missing], b = [missing, 4])
+
+        df = DataFrame(variable=["x", "x"], value=[missing, missing], id=[1,1])
+        @test_warn "Duplicate entries in unstack at row 2 for key 1 and variable x." unstack(df, :variable, :value)
+        @test_warn "Duplicate entries in unstack at row 2 for key 1 and variable x." unstack(df)
+    end
+
+    @testset "missing values in colkey" begin
+        df = DataFrame(id=[1, 1, 1, missing, missing, missing, 2, 2, 2],
+                       variable=["a", "b", missing, "a", "b", "missing", "a", "b", "missing"],
```

Comment:
Why `"missing"`? Doesn't seem to be tested specifically here.

Reply:
It is intentional. What I test:
* if I pass `missing` a warning should be printed;
* if I pass `"missing"` a column should be created with that name;

I added lines to test this.